src/pywifes.py

```diff
@@ -1892,7 +1892,7 @@ def wifes_slitlet_mef_ns(inimg, outimg_obj, outimg_sky,
         # create fits hdu for sky
         hdu_name = 'SCI%d' % (i+1)
         sky_hdu = pyfits.ImageHDU(sky_data, old_hdr, name=hdu_name)
-        sky_hdu.header.set('CCDSEC',  sky_dim_str)
+        sky_hdu.header.set('DETSEC',  sky_dim_str)
         sky_hdu.header.set('DATASEC', sky_dim_str)
         sky_hdu.header.set('TRIMSEC', sky_dim_str)
         # fix the exposure time!!
```

Comment:
Is the line below ok too? `exptime_true = float(old_hdr['NEXP'])*float(old_hdr['NSUBEXPS'])`. Doesn't refer to `SEXP`, does refer to `NSUBEXPS` though.